docs/source/development-guide/make/development.rst

```diff
@@ -13,6 +13,8 @@ These commands are used to control Augur's backend and frontend servers simultan
 
 ``make dev``
 -------------
+Note: If the above command doesn't work, try running ``make dev-start`` instead.
```

Comment:
I see that [`make dev`](https://github.com/chaoss/augur/blob/master/Makefile#L71) first stops the existing servers and turns them up again. However, for contributors running the project for the first time, it'd be nice to have a note mentioning about an alternate command if `make dev` fails. Let me know your thoughts on this.